docs/proposals/test-reports/pedestrian_tracking.md

```diff
@@ -0,0 +1,114 @@
+
+   $IDP=\frac{IDTP}{IDTP+IDFP}$
+
+   $IDR=\frac{IDTP}{IDTP+IDFN}$
+
+Reference
+
+[1] P. Dollar, C. Wojek, B. Schiele and P. Perona, "Pedestrian Detection: An Evaluation of the State of the Art," in IEEE Transactions on Pattern Analysis and Machine Intelligence, vol. 34, no. 4, pp. 743-761, April 2012, doi: 10.1109/TPAMI.2011.155.
+
+[2] Bernardin, K. & Stiefelhagen, R. Evaluating Multiple Object Tracking Performance: The CLEAR MOT Metrics. Image and Video Processing, 2008(1):1-10, 2008.
+[3] Ristani, E., Solera, F., Zou, R., Cucchiara, R. & Tomasi, C. Performance Measures and a Data Set for Multi-Target, Multi-Camera Tracking. In ECCV workshop on Benchmarking Multi-Target Tracking, 2016.
+[4] Jonathon Luiten, A.O. & Leibe, B. HOTA: A Higher Order Metric for Evaluating Multi-Object Tracking. International Journal of Computer Vision, 2020.
+[5] Li, Y., Huang, C. & Nevatia, R. Learning to associate: HybridBoosted multi-target tracker for crowded scene. In Proceedings of the IEEE Computer Society Conference on Computer Vision and Pattern Recognition, 2009.
+
+[6] Han, Xiaotian, et al. "MMPTRACK: Large-scale Densely Annotated Multi-camera Multiple People Tracking Benchmark." *arXiv preprint arXiv:2111.15157* (2021).
+
+#### Pedestrian Re-identification
+
+[Market-1501](https://openaccess.thecvf.com/content_iccv_2015/html/Zheng_Scalable_Person_Re-Identification_ICCV_2015_paper.html) is a benchmark datasets in the area of pedestrian detection and tracking.
+
+The following performance metrics are selected for re-identification.
+
+1. **Accuracy**
+
+2. **Precision**
+
+3. **Recall**
+
+4. **F1 score**
+
+5. **Rank-1**: first hit rate
+
+   $\operatorname{rank} 1=\frac{1}{\|Q\|} \sum_{q \in Q} \mathbb{1}\left(l_{q}, l_{1}^{q}\right)$
+
+6. **Rank-5**: top 5 hit rate
+
+7. **Rank-10**: top 10 hit rate
+
+8. **Rank-20**: top 20 hit rate
+
+Reference
+
+[1] Liang Zheng, Liyue Shen, Lu Tian, Shengjin Wang, Jingdong Wang, Qi Tian; Proceedings of the IEEE International Conference on Computer Vision (ICCV), 2015, pp. 1116-1124
+
+[2] Wei Li, Rui Zhao, Tong Xiao, Xiaogang Wang; Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition (CVPR), 2014, pp. 152-159
+
+### Framework
+![framework](images/pedestrian_tracking_framework.png)
+
+### Embedded in Ianvs
+
+This proposal is planned to be added to Ianvs in the form of a example.
+
+###  Test report
+
+Use [mdutils](https://github.com/didix21/mdutils) to generate markdown files and export PDFs. The test report provides datasets and their corresponding performance metrics, samples, etc.
+
+###  Detailed guidance documents
+
+```
+Re-ID Benchmark Documentation
+|- scenario
+|- algorithms
+    |-- pedestrian detection & tracking
+    |-- pedestrian re-identification
+|- datasets
+    |-- pedestrian detection & tracking
+    |-- pedestrian re-identification
+|- metrics
+    |-- pedestrian detection & tracking
+    |-- pedestrian re-identification
+|- usage
+|- reports
+```
```

Comment:
Re-ID Benchmarking files

ianvs
|- example
    |- re-id
        |- multi-edge inference bench
            |- testalgorithm
                |- mynet.yaml (named as the used algorithm)
            |- testenv
                |- testenv.yaml
                |- mAP.py (named as the used metric)
            |- benchmarking.yaml

Reply:
looks fine to me now